src/web/app/components/response-time-series-chart/response-time-series-chart.component.ts

```diff
@@ -19,30 +19,31 @@ export class ResponseTimeSeriesChartComponent implements OnInit {
 
   @Input()
   model: ResponseTimeSeriesChartModel = {
-    durationMinutes: 60,
-    intervalMinutes: 1,
+    durationMinutes: 1440,
```

Comment:
changed default duration to 1440 minutes (1 day) so that all data in the past day will be shown by default

Reply:
ok got it. Although i think 3hr is enough (else i'd need to populate a lot of data :p)